components/places/src/ffi.rs

```diff
@@ -168,20 +170,25 @@ impl PlacesApi {
     }
 
     fn places_history_import_from_fennec(&self, db_path: String) -> Result<String> {
-        let metrics = import_history(self, db_path.as_str())?;
+        let metrics = import_fennec_history(self, db_path.as_str())?;
         Ok(serde_json::to_string(&metrics)?)
     }
 
     fn places_bookmarks_import_from_fennec(&self, db_path: String) -> Result<String> {
-        let metrics = import_bookmarks(self, db_path.as_str())?;
+        let metrics = import_fennec_bookmarks(self, db_path.as_str())?;
         Ok(serde_json::to_string(&metrics)?)
     }
 
     fn places_bookmarks_import_from_ios(&self, db_path: String) -> Result<()> {
-        import_bookmarks(self, db_path.as_str())?;
```

Comment:
heh this function was wildly wrong. it called the `import_fennec_bookmarks` function to migration iOS bookmarks 😓 I fixed it here, but might just kill it

Reply:
🥴 Great. Well . . . an attempt was made to remove this logic in iOS two or more years ago and was only restored to make tests pass according to [this PR](https://github.com/mozilla-mobile/firefox-ios/pull/6393). So seems like this can die.